myapp/spec/system/tasks_spec.rb

```diff
@@ -135,9 +135,9 @@
 
   describe 'show tasks list ordered by specific column in ascending/descending order' do
     before do
-      create(:task, name: 'Task1', priority: 3, status: 2)
-      create(:task, name: 'Task2', priority: 1, status: 3)
-      create(:task, name: 'Task3', priority: 2, status: 1)
+      create(:task, name: 'Task1', priority: 3, status: 2, expired_date: '2023-06-30')
+      create(:task, name: 'Task2', priority: 1, status: 3, expired_date: '2024-06-30')
+      create(:task, name: 'Task3', priority: 2, status: 1, expired_date: '2023-07-30')
```

Comment:
exist priority: 3 and status: 3 ...?

```ruby
enum priority_type: {
    Low: 0,
    Medium: 1,
    High: 2,
  }
  enum status_type: {
    Todo: 0,
    Doing: 1,
    Done: 2,
  }
```

Reply:
it should not include 3!
I changed it in the comment below